### PR TITLE
Fix java dsl for createLogLevels 

### LINKED ISSUE #25313: Akka Streams Java DSL:  createLogLevels default args issue
https://github.com/akka/akka/blob/46b433b47daeabe438334aca70ac2cce7bade9fc/akka-stream/src/main/scala/akka/stream/Attributes.scala#L356

Trying to use `createLogLevels` from Java and setting one of the args as null, to get default values as the comment/docs suggests, will result in a compile error. The args are ints after erasure (since they are  case classes that extend AnyVal) so null is not allowed. In the code, the `Option(...).getOrElse(..)` check is fruitless since you'll never get a None. 

Not an expert in Java DSL but here's some suggested fixes:
- Provide a class called Default ..
```
.withAttributes(Attributes.createLogLevels(Logging.InfoLevel(), Logging.DefaultLevel(), Logging.DefaultLevel()))
```

- Provide "with" methods (`withLogLevelForOnElement(...)`, `withLogLevelForOnFinish(...)`, `withLogLevelForOnFailure(...)`)

### PR DESCRIPTION
**Pull request by sebastianharko**
Fix #25313

### CODE REVIEW

File: akka-stream/src/main/scala/akka/stream/Attributes.scala [hunk 1]
```diff
@@ -442,14 +447,19 @@ object ActorAttributes {
...
-      onElement = Option(onElement).getOrElse(Logging.DebugLevel),
-      onFinish = Option(onFinish).getOrElse(Logging.DebugLevel),
-      onFailure = Option(onFailure).getOrElse(Logging.ErrorLevel))
+    logLevels(onElement, onFinish, onFailure)
+
+  /**
+   * Java API
+   *
+   * Configures `log()` operator log-levels to be used when logging onElement.
+   * Logging a certain operation can be completely disabled by using [[LogLevels.Off]].
```
Comment (patriknw) [inline akka-stream/src/main/scala/akka/stream/Attributes.scala]: `[[Attributes#logLevelOff]]`

File: akka-stream/src/main/scala/akka/stream/Attributes.scala [hunk 2]
```diff
@@ -350,14 +350,19 @@ object Attributes {
...
+   *
+   */
+  def createLogLevels(onElement: Logging.LogLevel): Attributes =
+    logLevels(onElement)
 
   /**
    * Configures `log()` operator log-levels to be used when logging.
```
Comment (patriknw) [inline akka-stream/src/main/scala/akka/stream/Attributes.scala]: Next line should be `[[LogLevels.Off]]`

File: akka-stream/src/main/scala/akka/stream/Attributes.scala [hunk 3]
```diff
@@ -350,14 +350,19 @@ object Attributes {
...
-      onElement = Option(onElement).getOrElse(Logging.DebugLevel),
-      onFinish = Option(onFinish).getOrElse(Logging.DebugLevel),
-      onFailure = Option(onFailure).getOrElse(Logging.ErrorLevel))
+    logLevels(onElement, onFinish, onFailure)
+
+  /**
+   * Java API
+   *
+   * Configures `log()` operator log-levels to be used when logging onElement.
+   * Logging a certain operation can be completely disabled by using [[LogLevels.Off]].
```
Comment (patriknw) [inline akka-stream/src/main/scala/akka/stream/Attributes.scala]: `[[Attributes#logLevelOff]]`

**Review by patriknw — COMMENTED** [review]
some additional fixup of the scaladoc

File: akka-stream/src/main/scala/akka/stream/Attributes.scala (hunk 2 above)
Comment (sebastianharko) [reply akka-stream/src/main/scala/akka/stream/Attributes.scala]: @patriknw check now, i think all of them are fixed, hope I didn't miss anything ;-)

Comment (ktoso) [reply akka-stream/src/main/scala/akka/stream/Attributes.scala]: Looks good

File: akka-stream/src/main/scala/akka/stream/Attributes.scala [hunk 4]
```diff
@@ -440,16 +445,21 @@ object ActorAttributes {
    * Java API
    *
    * Configures `log()` operator log-levels to be used when logging.
-   * Logging a certain operation can be completely disabled by using [[LogLevels.Off]].
+   * Logging a certain operation can be completely disabled by using [[Attributes#logLevelOff]].
```
Comment (ktoso) [inline akka-stream/src/main/scala/akka/stream/Attributes.scala]: Should remain `Attributes.logLevelOff` I think

Comment (ktoso) [reply akka-stream/src/main/scala/akka/stream/Attributes.scala]: The name is but syntax isn't AFAIR, should remain `.`

Comment (ktoso) [reply akka-stream/src/main/scala/akka/stream/Attributes.scala]: Oh thanks, it seems then I was mis remembering the details here :) Thanks!

**Review by ktoso — APPROVED** [review]

**Pull request merged**